app/models/form_submission_attempt.rb

```diff
@@ -25,14 +25,15 @@ class FormSubmissionAttempt < ApplicationRecord
 
     event :fail do
       after do
+        form_type = form_submission.form_type
+        log_info = { form_submission_id:,
+                     benefits_intake_uuid: form_submission&.benefits_intake_uuid,
```

Comment:
Is the safe navigation operator necessary if there's a null constraint on `form_submission`?

Reply:
I dont want it to fail, no matter what, so I dont think it hurts here. Is it required, ideally no. 

I think it is on this and not the one on line 28 because `form_type` is added on create and a requirement/validation for creation. So it will never be nil. benefits intake uuid I think is a non-required arg, so there is potential it is not set, and that is not the end of the world, just means the record was created but no upload uuid was able to be gotten.

Reply:
`form_submission` doesn't use the safe navigation operator in the other usages in the lines above and below this, just mentioning for consistency sake might want to make it all one way or the other